noseiquela_orm/entity.py

```diff
@@ -1,17 +1,20 @@
-from typing import Any, Callable, Dict, Tuple
+from typing import Any, Callable, Dict, List, Optional, Tuple, TYPE_CHECKING
 from google.cloud.datastore.entity import Entity as GoogleEntity
 
 from .client import DataStoreClient
 from .query import Query
 from .properties import BaseProperty, BaseKey, KeyProperty
 from .utils.case_style import CaseStyle
 
+if TYPE_CHECKING:
+    from .properties import ParentKey
+
 
 class EntityMetaClass(type):
     def __init__(self, name: str, bases: Tuple, attrs: Dict) -> None:
         super().__init__(name, bases, attrs)
 
-        self.kind = attrs.get("__kind__") or name
+        self.kind: str = attrs.get("__kind__") or name
```

Comment:
sometimes, on assignment statements, i added the types to help us (and helps the IDEs also) to understand what we have in each property/attribute